### PR TITLE
Add bool to levelsettings whether to show infection status or not

### PR DESCRIPTION
**Pull request by holke**
This makes it more feasible for levels different then gethome
to have their infection status shown or not

Also got rid of magic number on NPC creation.

### CODE REVIEW

**Review by maccxs — CHANGES_REQUESTED** [review]
A good little change
ShowInfectinoStatus --> should be ShowInfectionStatus
In InfectionControl.cs we set the day lengt. Maybe this should also be controlled over levelSettings?
DayLength  = 1f is standard for sandbox, in the other levels I set something like 100f.

**Comment by holke** [comment]
I strongly disagree. Instead, InfectionControl should better be called InfectinoControl!

File: YouVsVirus/Assets/Scripts/Components/HumanBase.cs [hunk 1]
```diff
@@ -135,7 +135,7 @@ public virtual void SetCondition(int condition)
             }
 
             // in Levelgethome we want all the smileys to stay the same except for the player
-            if(LevelSettings.GetActiveSceneName() != "YouVsVirus_Levelgethome" || this.tag == "Player" )
+            if(LevelSettings.GetActiveLevelSettings().ShowInfectinoStatus == true || this.tag == "Player" )
```
Comment (holke) [inline YouVsVirus/Assets/Scripts/Components/HumanBase.cs]:
```suggestion
            if(LevelSettings.GetActiveLevelSettings().ShowInfectionStatus == true || this.tag == "Player" )
```

File: YouVsVirus/Assets/Scripts/Components/LevelSettings.cs [hunk 2]
```diff
@@ -26,6 +26,11 @@ public class LevelSettings : MonoBehaviour
     /// </summary>
     public float SocialDistancingFactor = 50f;
 
+    /// <summary>
+    /// Do we show the infection status of the smileys in this level or not.
+    /// </summary>
+    public bool ShowInfectinoStatus = true;
```
Comment (holke) [inline YouVsVirus/Assets/Scripts/Components/LevelSettings.cs]:
```suggestion
    public bool ShowInfectionStatus = true;
```

File: YouVsVirus/Assets/Scripts/levelgethome/CreatePopLevelgethome.cs [hunk 3]
```diff
@@ -64,6 +64,8 @@ void Start()
             // get active level settings - the get home scene always has 50% social distancing
             LevelSettings.GetActiveLevelSettings().SocialDistancingFactor = 18;
             LevelSettings.GetActiveLevelSettings().NumberOfNPCs = npcNumber;
+            // We do not show the infection status in this level
+            LevelSettings.GetActiveLevelSettings().ShowInfectinoStatus = false;
```
Comment (holke) [inline YouVsVirus/Assets/Scripts/levelgethome/CreatePopLevelgethome.cs]:
```suggestion
            LevelSettings.GetActiveLevelSettings().ShowInfectionStatus = false;
```

**Comment by holke** [comment]
I moved the dayLength to levelsettings.

**Pull request merged**